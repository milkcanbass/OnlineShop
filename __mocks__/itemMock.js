export const mockItem = {
  imageUrl: 'url',
  price: 0,
  name: 'name',
  quantity: 0,
};
