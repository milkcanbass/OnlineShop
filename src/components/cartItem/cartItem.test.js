import React from 'react';
import { shallow } from 'enzyme';
import CartItem from './cartItem.component';
import { mockItem } from '../../../__mocks__/itemMock';

it('takeSnapshot', () => {
  expect(shallow(<CartItem item={mockItem} />)).toMatchSnapshot();
});
