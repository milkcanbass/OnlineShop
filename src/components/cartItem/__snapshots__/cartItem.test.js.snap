// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`takeSnapshot 1`] = `
<div
  className="cartItem"
>
  <img
    alt="itemImage"
    src="url"
  />
  <div
    className="itemDetails"
  >
    <span
      className="name"
    >
      name
    </span>
    <span
      className="price"
    >
      $
      0
      x
      0
    </span>
  </div>
</div>
`;
